CHANGELOG.md

```diff
@@ -36,4 +37,4 @@ First stable release: no breaking changes will be made in the 1.x series.
 
 ## v1.0.0-rc1 (2017-06-26)
 
-- **[Breaking]** `Provider` interface was trimmed down to 2 methods: `Name` and `Get`
\ No newline at end of file
+- **[Breaking]** `Provider` interface was trimmed down to 2 methods: `Name` and `Get`
```

Comment:
?

Reply:
Github's doing a bad job displaying this - the file didn't have a newline at the end.